src/test/regress/multi_schedule

```diff
@@ -134,3 +134,5 @@ test: check_mx
 test: generated_identity
 test: drop_database
 test: check_cluster_state
+
+test: function_with_case_when
```

Comment:
can you move this test above line 125, which contains `ensure_no_intermediate_data_leak`?

Reply:
done